plexpy/webserve.py

```diff
@@ -2228,6 +2228,14 @@ def random_arnold_quotes(self, **kwargs):
                       'I\'m a cop you idiot!',
                       'Come with me if you want to live.',
                       'Who is your daddy and what does he do?'
+					  'Oh, cookies! I can\'t wait to toss them.'
```

Comment:
Use 4 spaces instead of a "tab".

There also needs to be a comma at the end of each line (except the last one).
````
'Oh, cookies! I can\'t wait to toss them.',
'Can you hurry up. My horse is getting tired.',
etc...
```